ipatests/test_webui/test_host.py

```diff
@@ -407,11 +435,353 @@ def test_indirect_associations(self):
         self.assert_indirect_record(hbac.RULE_PKEY, ENTITY, 'memberof_hbacrule')
         self.assert_indirect_record(sudo.RULE_PKEY, ENTITY, 'memberof_sudorule')
 
-        ## cleanup
-        ## -------
+        # cleanup
+        # -------
         self.delete(ENTITY, [self.data])
         self.delete(hostgroup.ENTITY, [hostgroup.DATA, hostgroup.DATA2])
         self.delete(netgroup.ENTITY, [netgroup.DATA])
         self.delete(rbac.ROLE_ENTITY, [rbac.ROLE_DATA])
         self.delete(hbac.RULE_ENTITY, [hbac.RULE_DATA])
         self.delete(sudo.RULE_ENTITY, [sudo.RULE_DATA])
+
+    @screenshot
+    def test_buttons(self):
+        """Test buttons"""
+        self.init_app()
+        actions = ActionChains(self.driver)
```

Comment:
let's move it to the usage of `actions` as there is only one. So that it is better editable in future.

Reply:
it is used more that one time in function so better have it as init